src/containers/WrappedPositionsList/WrappedPositionsList.tsx

```diff
@@ -34,6 +34,12 @@ export const WrappedPositionsList: React.FC = () => {
     dispatch(actions.setLastPage(page))
   }
 
+  useEffect(() => {
+    if (lastPage > Math.ceil(list.length / 5)) {
```

Comment:
Consider the expression: `Math.ceil(list.length / 5)`

Now, what if the number of positions on the list changes, or users are given the option to choose that number from UI? It seems there could be potential issues, especially when trying to locate all the elements dependent on the number of items displayed on one page (currently 5). Therefore, it's beneficial to centralize this information as much as possible and establish singular sources.